test/Models/TerriaSpec.ts

```diff
@@ -141,6 +142,45 @@ describe("Terria", function() {
       jasmine.Ajax.uninstall();
     });
 
+    it("works with initializationUrls and initFragmentPaths", async function() {
+      expect(terria.initSources.length).toEqual(0);
+
+      jasmine.Ajax.stubRequest("path/to/config/configUrl.json").andReturn({
+        responseText: JSON.stringify({
+          initializationUrls: ["something"],
+          parameters: {
+            applicationUrl: "https://application.url/",
+            initFragmentPaths: [
+              "path/to/init/",
+              "https://hostname.com/some/other/path/"
+            ]
+          }
+        })
+      });
+
+      await terria.start({
+        configUrl: `path/to/config/configUrl.json`,
+        i18nOptions
+      });
+
+      expect(terria.initSources.length).toEqual(1);
+
+      const initSource = terria.initSources[0];
+      expect(isInitFromOptions(initSource)).toBeTruthy();
+
+      if (!isInitFromOptions(initSource))
+        throw "Init source is not from options";
+
+      expect(
+        initSource.options.map(source =>
+          isInitFromUrl(source) ? source.initUrl : ""
+        )
+      ).toEqual([
+        "path/to/config/path/to/init/something.json",
```

Comment:
This looks odd to me. Shouldn't it be "path/to/init/something.json"? Need to document how a path is constructed.

Reply:
I think, the initFragmentPath should be relative to baseUrl specified by configUrl. Then by setting
```
initFragmentPaths: [
              "../init/",
              "https://hostname.com/some/other/path/"
            ]
```
the path will be constructed as "path/to/init/something.json".

Reply:
I have added comments to how paths are constructed